### ISSUE TITLE
Error on nest_asyncio.apply()

### ISSUE THREAD

**Issue by jhogg11** [spyder-ide/spyder#17875]
## Problem Description

I'm trying to set up a new Mac and get some existing projects running.  Specifically, I'm trying to use `nest_asyncio` in Spyder and getting an error on `nest_asyncio.apply()` as well as inability to restart the kernel after applying.

### What steps reproduce the problem?

    import asyncio
    import nest_asyncio
    nest_asyncio.apply()

### Paste Traceback/Error Below (if applicable)

And I get the following error message:

```
[SpyderKernelApp] ERROR | Exception in message handler:
Traceback (most recent call last):
  File "/Users/me/opt/anaconda3/lib/python3.9/site-packages/spyder_kernels/comms/frontendcomm.py", line 164, in poll_one
    asyncio.run(handler(out_stream, ident, msg))
  File "/Users/me/opt/anaconda3/lib/python3.9/site-packages/nest_asyncio.py", line 33, in run
    task = asyncio.ensure_future(main)
  File "/Users/me/opt/anaconda3/lib/python3.9/asyncio/tasks.py", line 677, in ensure_future
    raise TypeError('An asyncio.Future, a coroutine or an awaitable is '
TypeError: An asyncio.Future, a coroutine or an awaitable is required
[SpyderKernelApp] ERROR | Exception in message handler:
Traceback (most recent call last):
  File "/Users/me/opt/anaconda3/lib/python3.9/site-packages/spyder_kernels/comms/frontendcomm.py", line 164, in poll_one
    asyncio.run(handler(out_stream, ident, msg))
  File "/Users/me/opt/anaconda3/lib/python3.9/site-packages/nest_asyncio.py", line 33, in run
    task = asyncio.ensure_future(main)
  File "/Users/me/opt/anaconda3/lib/python3.9/asyncio/tasks.py", line 677, in ensure_future
    raise TypeError('An asyncio.Future, a coroutine or an awaitable is '
TypeError: An asyncio.Future, a coroutine or an awaitable is required
```

The error appears to be unique to Spyder.  I don't see any errors when I run it in the terminal, Jupyter notebook, QT console, etc.

## Versions

* Spyder version:  5.1.5
* Python version:  3.9.12

**Comment by ccordoba12** [comment]
Hey @jhogg11, thanks for reporting. I think this error is fixed in our latest version (5.3.0).

Since it's still not available in Anaconda, you have two options:

1. Use our Mac installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:

        conda create -n spyder-cf -c conda-forge spyder
        conda activate spyder-cf
        spyder

Let us know if that works for you.

**Comment by jhogg11** [comment]
Thanks for the quick response.  Nest_asyncio seems to be working after using the Mac installer.

Any idea on when 5.3.0 will be available in Anaconda?

**Comment by ccordoba12** [comment]
Nop, we don't have any control over that.

I'm going to close this issue because the bug is solved on our side.

**Comment by jhogg11** [comment]
> Hey @jhogg11, thanks for reporting. I think this error is fixed in our latest version (5.3.0).
> 
> Since it's still not available in Anaconda, you have two options:
> 
> 1. Use our Mac installer, which you can find [here](https://github.com/spyder-ide/spyder/releases).
> 2. Create a new environment with conda-forge packages. For that, please close Spyder, open the Anaconda Prompt and run the following commands there:
>    ```
>     conda create -n spyder-cf -c conda-forge spyder
>     conda activate spyder-cf
>     spyder
>    ```
> 
> Let us know if that works for you.

One other question - is it currently possible to install Spyder on an arm64 environment?  I want to run tensorflow, which I understand requires Miniforge3 arm64, however, it seems I can't get both tensorflow and Spyder in the same environment.  The error I'm getting when trying to open Spyder (in a new conda environment) is:

`No module named 'PyQt5.QtWebKitWidgets'`

But then I run into other issues trying with PyQt5.  Are you aware of this issue or any way to get around it?

**Comment by jhogg11** [comment]
Found the answer.  Instead of installing `pyqtwebengine` through pip, it needed to be installed through conda-forge:

`conda install -c conda-forge pyqtwebengine`

From one of the comments here:  https://stackoverflow.com/questions/71580652/how-to-solve-error-while-installing-pyqtwebengine

@ccordoba12 - Thanks again for your help.
